src/sql/base/browser/ui/table/media/table.css

```diff
@@ -176,9 +176,8 @@
 	align-items: center;
 }
 
-.slick-icon-cell,
 .slick-button-cell {
-	padding: 0px !important;
+	padding: 5px !important;
```

Comment:
leave some padding space so that the focus indicator of the button is not overlapped with the cell's border. see screenshot.